### PR TITLE
Update S3 bucket documentation

### PR DESCRIPTION
**Pull request by bradyab**
I just went through this process and it took me ~5x longer than it should have because there were critical details missing and the docs have diverged from both the landscape and Digital Ocean UIs. Here we address those issues

### CODE REVIEW

File: content/manual/os/s3.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Click the green Create button at the top of the project page, then click Spaces.
 
 Make the following selections:
 - Select the datacenter region closest to you
-- Restrict File Listing (leave this as the default)
+- Enable CDN if you prefer
```
Comment (bradyab) [inline content/manual/os/s3.md:25]: `restrict file listing` is no longer an option. `enable CDN` is

File: content/manual/os/s3.md [hunk 2]
```diff
@@ -35,23 +35,25 @@ Finally you'll need to create an API key so you can configure your ship to have
 
 Click API in the bottom left of the DigitalOcean navigation bar on the left side of the webpage.
 
-Generate a new token, you'll need to copy the access key and secret. Don't navigate away from this page as the secret will only be shown when it is initially created.
+Generate a new `Spaces access key`. Note that this is distinct from a `Personal access token`. You'll need to copy the access key and secret. Don't navigate away from this page as the secret will only be shown when it is initially created.
```
Comment (bradyab) [inline content/manual/os/s3.md:38]: both of these options are on the same page and they can be easily confused
Screen Shot 2022-12-31 at 10 45 12 AM

File: content/manual/os/s3.md [hunk 3]
```diff
@@ -35,23 +35,25 @@ Finally you'll need to create an API key so you can configure your ship to have
...
-The Access Key ID and Secret Access Key is the API key we generated. Put each one in the correct field and save.
+The Access Key ID and Secret Access Key are the API keys we just generated. Put each one in the correct field and save.
 
-The S3 Bucket is the name of your digital ocean space. Once you've added that you'll also need to click "Make Active" from the drop down menu.
+The Bucket Name is the name of your digital ocean space. Once you've added that you'll also need to click "Save" at the bottom of the form.
 
 Once all of that's complete you should be good to go! Test that things are working by attempting to attach or send and image via chat. It should work. You can manage your images by logging into Digital Ocean and looking at the folder in your newly created space. You can use your same space for multiple urbit IDs by creating new API keys. Each ID will create a new directory within that space using its name by default.
 
-To manage your S3 inventory on your urbit, install Silo: web+urbitgraph://~dister-nocsyx-lassul/silo
+To manage your S3 inventory on your urbit, install Silo: web+urbitgraph://~dister-nocsyx-lassul/silo (Warning: still in Beta)
```
Comment (bradyab) [inline content/manual/os/s3.md:54]: silo was not working for me despite still being able to upload from Groups with same s3 config

**Comment by bradyab** [comment]
> @bradyab is attempting to deploy a commit to the **urbit** Team on [Vercel](https://vercel.com).
> 
> A member of the Team first needs to [authorize it](https://vercel.com/git/authorize?team=urbit&type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%220fc936ffd4150b731ed9823f9b6efc4d5583b25f%22%7D%2C%22id%22%3A%22QmbSnSr4uoQZqnLAKyabr7mppzrndNzG2VoZJccxEQ48ez%22%2C%22org%22%3A%22urbit%22%2C%22prId%22%3A67%2C%22repo%22%3A%22operators.urbit.org%22%7D).

@sigilante can you authorize this?

**Comment by sigilante** [comment]
I am not sure—none of my attempts seem to have made a difference.  @gordoncc?

**Comment by gordoncc** [comment]
@sigilante – I don't have admin access on Vercel to approve deployments either.

@bradyab – thanks for the updates on this guide.  In the future you can just tag one of us and we'll review and merge.

**Pull request merged**